.github/workflows/php.yml

```diff
@@ -0,0 +1,46 @@
+name: PHP Composer
```

Comment:
Rename to continuous-integration.yml; Reference orm ci for hints
https://github.com/laravel-doctrine/orm/blob/3.1.x/.github/workflows/continuous-integration.yml